### PR TITLE
Add blog post about updates with WebAuthn and MFA

### PR DESCRIPTION
**Pull request by jenshenny**
Part of: https://github.com/rubygems/rubygems.org/issues/3800

From the last blog post, we mentioned that we're still working towards having WebAuthn as a valid MFA factor. As that is approaching completion, here's is the follow up blog post to announce these changes and to encourage users to enable security devices and MFA to their accounts.

This is currently blocked on the updates to the documentation https://github.com/rubygems/guides/pull/339

### CODE REVIEW

File: _posts/2023-07-24-level-up-using-security-devices.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Level up your RubyGems Account Security with Security Devices
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+
+In a [previous blog post](https://blog.rubygems.org/2023/04/11/security-device-cli-support.html), we introduced the latest updates to our security features, including CLI support for security devices and future plans for the feature.
+
+We are excited to announce that WebAuthn is fully integrated and is recognized to be the preferred multi-factor authentication (MFA) factor in RubyGems. When users register a security device, they will now be provided with recovery codes and be able to configure their [multi-factor authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) without needing to enable time-based MFA.
```
[the remark this answers is not in the document]
Comment (ericherscovich) [reply _posts/2023-07-24-level-up-using-security-devices.md:10]: Perhaps you could be more specific with 'time-based MFA' by saying 'time-based OTP'?

**Review by ericherscovich — COMMENTED** [review]
Two small things for you to ponder about, but other than that looks great!

File: _posts/2023-07-24-level-up-using-security-devices.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### Why use a Security Device?
+Security devices add an extra layer of protection beyond passwords. It utilizes the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which employs public-key cryptography to mitigate the risk of password-related attacks such as phishing or credential stuffing.
+
+They provide better security and convenience compared to their time-based one time password counterparts, making them a preferred choice for authentication.
+
+Time-based codes from an authentication app can be intercepted or manipulated by phishing attacks. Security devices, on the other hand, eliminates the need of code inputs while providing a stronger protection against phishing attempts as they require the security key or the user (for biometric devices) to be present.
```
Comment (george-ma) [inline _posts/2023-07-24-level-up-using-security-devices.md:17]:
```suggestion
While time-based codes from an authentication app can be intercepted or manipulated by phishing attacks, security devices eliminate the need for code inputs. They provide stronger protection against phishing attempts as they require the security key or the user (for biometric devices) to be present. This eliminates the risks associated with code-based authentication, ensuring a more robust defence against unauthorized attacks.
```

File: _posts/2023-07-24-level-up-using-security-devices.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+
+In a [previous blog post](https://blog.rubygems.org/2023/04/11/security-device-cli-support.html), we introduced the latest updates to our security features, including CLI support for security devices and future plans for the feature.
+
+We are excited to announce that WebAuthn is fully integrated and is recognized to be the preferred choice for multi-factor authentication (MFA). When users register a security device, they will now be provided with recovery codes and be able to configure their [multi-factor authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) without needing to enable time-based one time password (OTP) MFA.
+
+### Why use a Security Device?
+Security devices add an extra layer of protection beyond passwords. It utilizes the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which employs public-key cryptography to mitigate the risk of password-related attacks such as phishing or credential stuffing.
```
Comment (george-ma) [inline _posts/2023-07-24-level-up-using-security-devices.md:13]:
```suggestion
Security devices offer an additional layer of protection that goes beyond passwords. By leveraging the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which utilizes public-key cryptography, they effectively mitigate the risk of password-related attacks, such as phishing or credential stuffing.
```

File: _posts/2023-07-24-level-up-using-security-devices.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+They provide better security and convenience compared to their time-based one time password counterparts, making them a preferred choice for authentication.
```
Comment (george-ma) [inline _posts/2023-07-24-level-up-using-security-devices.md:15]:
```suggestion
These devices not only provide enhanced security but also offer greater convenience compared to traditional time-based one-time password methods, making them a preferred choice for authentication.
```

**Comment by george-ma** [comment]
Some small changes to make it flow better for the _why use security devices?_ section, but otherwise LGTM! 🙌🏻

File: _posts/2023-07-24-level-up-using-security-devices.md (hunk 4 above)
Comment (jenshenny) [reply _posts/2023-07-24-level-up-using-security-devices.md]: I'm not too sure about using `traditional` to describe OTP, so I omitted that but I took the rest of the suggested wording 👍

**Review by segiddins — APPROVED** [review]

File: _posts/2023-07-24-level-up-using-security-devices.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+### Why use a Security Device?
+Security devices offer an additional layer of protection that goes beyond passwords. By leveraging the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which utilizes public-key cryptography, they effectively mitigate the risk of password-related attacks. This includes phishing or credential stuffing.
+
+These devices not only provide enhanced security but also offer greater convenience compared to their time-based one time password counterparts, making them a preferred choice for authentication.
+
+While time-based codes from an authentication app can be intercepted or manipulated by phishing attacks, security devices eliminate the need for code inputs. They provide stronger protection against phishing attempts as they require the security key or the user (for biometric devices) to be present. This eliminates the risks associated with code-based authentication, ensuring a more robust defence against unauthorized attacks.
+
+### Why use a Security Device?
```
Comment (simi) [inline _posts/2023-07-24-level-up-using-security-devices.md:19]: How?

Comment (jenshenny) [reply _posts/2023-07-24-level-up-using-security-devices.md]: Yep 🙈 copy paste mistake when I was copying over from a document. Thanks for catching it 🙏

**Comment by ericherscovich** [comment]
Updated the date of the blog to be today. Should be good to merge 👍.

**Pull request merged**